lighthouse-core/audits/accessibility/valid-lang.js

```diff
@@ -22,7 +22,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Specifying a valid [BCP 47 language](https://www.w3.org/International/questions/qa-choosing-language-tags#question) ' +
       'on elements helps ensure that text is pronounced correctly by a screen reader. ' +
-      '[Learn more](https://dequeuniversity.com/rules/axe/4.4/valid-lang).',
+      '[Learn more about `lang` attribute](https://dequeuniversity.com/rules/axe/4.4/valid-lang).',
```

Comment:
plural/or missing the

would be good to pick one style of this. learn more about the X attribute – or more learn about X attributes. My weak preference is for the former

Reply:
Agree, former is better. I seem to have made this pattern all over; will check for this before I proceed.